package.json

```diff
@@ -82,31 +82,31 @@
     "react-native": "0.69.11",
     "react-native-background-timer": "2.4.1",
     "react-native-calendar-events": "2.2.0",
-    "react-native-callstats": "3.73.7",
+    "react-native-callstats": "3.73.22",
     "react-native-default-preference": "1.4.4",
-    "react-native-device-info": "8.4.8",
-    "react-native-dialog": "https://github.com/jitsi/react-native-dialog/releases/download/v9.2.2-jitsi.1/react-native-dialog-9.2.2.tgz",
+    "react-native-device-info": "10.9.0",
+    "react-native-dialog": "9.3.0",
     "react-native-gesture-handler": "2.9.0",
-    "react-native-get-random-values": "1.7.2",
+    "react-native-get-random-values": "1.9.0",
     "react-native-immersive-mode": "2.0.1",
     "react-native-keep-awake": "4.0.0",
     "react-native-orientation-locker": "1.5.0",
-    "react-native-pager-view": "5.4.9",
-    "react-native-paper": "5.1.2",
-    "react-native-performance": "2.1.0",
-    "react-native-safe-area-context": "4.6.4",
-    "react-native-screens": "3.22.0",
-    "react-native-sound": "0.11.1",
+    "react-native-pager-view": "6.2.0",
+    "react-native-paper": "5.10.3",
+    "react-native-performance": "5.0.0",
+    "react-native-safe-area-context": "4.7.1",
+    "react-native-screens": "3.24.0",
+    "react-native-sound": "0.11.2",
     "react-native-splash-screen": "3.3.0",
-    "react-native-svg": "12.4.3",
-    "react-native-svg-transformer": "1.0.0",
-    "react-native-tab-view": "3.1.1",
-    "react-native-url-polyfill": "1.3.0",
+    "react-native-svg": "13.13.0",
+    "react-native-svg-transformer": "1.1.0",
+    "react-native-tab-view": "3.5.2",
+    "react-native-url-polyfill": "2.0.0",
     "react-native-video": "https://git@github.com/react-native-video/react-native-video#7c48ae7c8544b2b537fb60194e9620b9fcceae52",
```

Comment:
Can you pl check? I think this commit is alpha.1, they may have updated.

Reply:
https://github.com/react-native-video/react-native-video/commit/7c48ae7c8544b2b537fb60194e9620b9fcceae52 it's alpha.2 if I am not mistaken. Should we go to alpha.7. That is the latest, right?

Reply:
I suppose so yeah.